### PR TITLE
Add EN Blog: Serverless vs FaaS

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@kubesphere.io>

/assign @JiannaLiu @FeynmanZhou 
Hi, I uploaded the blog of serverless and FaaS. Please take a look, thanks. -> [Preview link](https://deploy-preview-2327--kubesphere-v3-1.netlify.app/blogs/serverless-vs-faas/).

### CODE REVIEW

File: content/en/blogs/serverless-vs-faas.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- Google Cloud Functions
+- Azure Functions
+
+Although many cloud providers have their serverless or FaaS solutions ready for developers, it is possible to face the issue of vendor lock-in after you run your business functions on a single platform.
+
+### Open-source serverless/FaaS alternatives
+
+In recent years, many outstanding open-source projects (such as [KEDA](https://keda.sh/), [Dapr](https://dapr.io/), [Cloud Native Buildpacks (CNB)](https://buildpacks.io/), [Tekton](https://tekton.dev/), and [Shipwright](https://shipwright.io/)) have emerged in the cloud-native Serverless-related fields. It is not easy to select the appropriate tools and put them together to build your own serverless solution.
+
+Thus, [OpenFunction](https://github.com/OpenFunction/OpenFunction/) comes as a cloud-native open-source FaaS platform aiming to let you focus on your business logic without having to maintain the underlying runtime environment and infrastructure. It can be easily deployed to your Kubernetes cluster. You can generate event-driven and dynamically scaling serverless workloads by simply submitting business-related source code in the form of functions.
```
Comment (JiannaLiu) [inline content/en/blogs/serverless-vs-faas.md:92]: We can recommend readers here to learn more about the use case of OpenFunction through https://kubesphere.io/blogs/faas-openfunction/.

Also please add a hyperlink to 'FaaS platform' with https://openfunction.dev/

Comment (Felixnoo) [reply content/en/blogs/serverless-vs-faas.md]: Thanks for your ideas. Updated.

**Comment by FeynmanZhou** [comment]
@Felixnoo ,
As the CNCF Serverless Whitepaper mentioned below, Serverless contains FaaS and BaaS. Is it appropriate to explain Serverless and FaaS as opposing concepts?
image

**Comment by Felixnoo** [comment]
> @Felixnoo , As the CNCF Serverless Whitepaper mentioned below, Serverless contains FaaS and BaaS. Is it appropriate to explain Serverless and FaaS as opposing concepts? image

This article tries to introduce serverless and FaaS respectively, rather than explaining them as two "opposing" concepts. There are no such words in this article to estrange FaaS from serverless.
BTW, A containing B doesn't necessarily mean that there is no difference between A and B. A and B have their differences doesn't rule out the possibility of A containing B.
You can see other articles share the same idea with this one. See [this example](https://thenewstack.io/add-it-up-serverless-faas/).

**Comment by FeynmanZhou** [comment]
> > @Felixnoo , As the CNCF Serverless Whitepaper mentioned below, Serverless contains FaaS and BaaS. Is it appropriate to explain Serverless and FaaS as opposing concepts? image
> 
> This article tries to introduce serverless and FaaS respectively, rather than explaining them as two "opposing" concepts. There are no such words in this article to estrange FaaS from serverless. BTW, A containing B doesn't necessarily mean that there is no difference between A and B. A and B have their differences doesn't rule out the possibility of A containing B. You can see other articles share the same idea with this one. See [this example](https://thenewstack.io/add-it-up-serverless-faas/).

Make sense, it's reasonable

**Pull request merged**